app/server/src/features/accounts/methods.test.ts

```diff
@@ -0,0 +1,349 @@
+import bcrypt from 'bcryptjs';
+import { register, findUserById, findOrgsByUserId, registerSelf, loginWithPassword } from './methods';
+import { prismaMock } from '../../../mocks/prisma/singleton';
+import * as jwt from '@local/lib/jwt';
+import { toGlobalId } from '../utils';
+import { faker } from '@faker-js/faker';
+
+jest.mock('@local/lib/jwt', () => ({
+    ...jest.requireActual('@local/lib/jwt'), // import and retain the original functionalities
+}));
```

Comment:
I'm not sure jamella was the one that wrote that in [this commit ](https://github.com/c2c-project/prytaneum/pull/124/commits/d48101663b13bc2dd4a2f1b8e03e0d7da3342dcf). I moved over and updated code from an old pr where she set it up since I had too much trouble rebasing it.

Reply:
It's possible it's unnecessary, I just tested it without that code and the tests still passed.

Reply:
Ack. Yeah, it seems, I'm not sure what's going on there.  I get what the code does -- it mocks the existing file with the actual file, which seems weird.  There may be a reason, like it allows for more easily spying, but again seems a bit odd.